Add `osu!taiko World Cup 2024: Round of 32 Recap` news post (#11175)
ppy/osu-wiki: news/2024/2024-03-27-osutaiko-world-cup-2024-round-of-32-recap.md

## Match highlights

### ::{ flag=MY }:: Malaysia vs. ::{ flag=CH }:: Switzerland

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2099153913&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Malaysia's local tournament scene ([OTMT](/wiki/Tournaments/OTMT/2022), [GTMS](https://osu.ppy.sh/community/forums/topics/1726861)) has proven itself to be highly beneficial for its players. Even though only ::{ flag=MY }:: [Crabcow](https://osu.ppy.sh/users/9755504) remains from their 2023 team, Malaysia managed to secure a higher seeding than ever before, landing at #11 with consistent placements in all but Double Time. Their opponent, Switzerland, have introduced ::{ flag=CH }:: [NekoTowel](https://osu.ppy.sh/users/14838605) to their otherwise unchanged roster. Looking at Qualifiers scores, Malaysia and Switzerland's preferences seem perfectly matched (albeit with different scores...), making this a bona fide mirror match between the 11th and 22nd seed teams.

> Switzerland decides there's no better opener than [KISIDA KYODAN & THE AKEBOSI ROCKETS - Shinkuu no Melancholy](https://osu.ppy.sh/beatmapsets/2152054#taiko/4534311), and it *almost* pays off, though a 99.88% Full Combo from ::{ flag=MY }:: [K0rd31HP](https://osu.ppy.sh/users/15231510) push Malaysia towards an immediate break point. Malaysia snags another close victory through [xi - Garyou Tensei](https://osu.ppy.sh/beatmapsets/2152045#taiko/4534285), but the momentum seems to get to their head as Switzerland grabs their first point on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) thanks to their opponents over-modding.

Reviewer: ```suggestion
Switzerland decides there's no better opener than [KISIDA KYODAN & THE AKEBOSI ROCKETS - Shinkuu no Melancholy](https://osu.ppy.sh/beatmapsets/2152054#taiko/4534311), and it *almost* pays off. However, a 99.88% Full Combo from ::{ flag=MY }:: [K0rd31HP](https://osu.ppy.sh/users/15231510) pushes Malaysia towards an immediate break point. Malaysia snags another close victory through [xi - Garyou Tensei](https://osu.ppy.sh/beatmapsets/2152045#taiko/4534285), but the momentum seems to get to their head as Switzerland grabs their first point on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) thanks to their opponents over-modding.
```

> The teams continue to trade blows throughout the remaining picks, but just as all the viewers collectively scream in excitement for a tiebreaker, Malaysia narrowly avoids it through a win on [REDALiCE feat. Nomiya Ayumi - Colors](https://osu.ppy.sh/beatmapsets/1578929#taiko/3223966) and leave their shadow clones behind as they move on to the Round of 16 Winners' Bracket.

Reviewer: ```suggestion
The teams continue to trade blows throughout the remaining picks, but just as all the viewers collectively scream in excitement for a tiebreaker, the Malaysians narrowly avoid it through a win on [REDALiCE feat. Nomiya Ayumi - Colors](https://osu.ppy.sh/beatmapsets/1578929#taiko/3223966) and leave their shadow clones behind as they move on to the Round of 16 winners' bracket.
```

### ::{ flag=TR }:: Turkey vs. ::{ flag=PL }:: Poland

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2099334355&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Team Turkey, spearheaded by ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550), returns after a 2023 hiatus alongside TWC newcomers ::{ flag=TR }:: [magev](https://osu.ppy.sh/users/13892153), ::{ flag=TR }:: [Zmor0133](https://osu.ppy.sh/users/6419257) and ::{ flag=TR }:: [Ryukishi](https://osu.ppy.sh/users/9045874). Their placements on the Qualifiers leaderboards make a nice V-shape, clearly illustrating that they're not too fond of Hidden. Their opponents, Team Poland, sport their previous roster as well as introducing a new member in the way of ::{ flag=PL }:: [Dynutka](https://osu.ppy.sh/users/5947521). Poland's Qualifiers scores hover closely around Turkey's, even beating them out on the aforementioned Hidden.

Reviewer: ```suggestion
Team Turkey, spearheaded by ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550), returns after a 2023 hiatus alongside TWC newcomers ::{ flag=TR }:: [magev](https://osu.ppy.sh/users/13892153), ::{ flag=TR }:: [Zmor0133](https://osu.ppy.sh/users/6419257) and ::{ flag=TR }:: [Ryukishi](https://osu.ppy.sh/users/9045874). Their placements on the Qualifiers leaderboards make a nice V-shape, clearly illustrating that they're not too fond of Hidden. Their opponents, Team Poland, sport their previous roster as well as introducing a new member in the way of ::{ flag=PL }:: [Dynutka](https://osu.ppy.sh/users/5947521). Poland's Qualifier scores hover closely around Turkey's, even beating them out on the aforementioned Hidden.
```

> Turkey opens by playing it safe, testing the waters with [KISIDA KYODAN & THE AKEBOSI ROCKETS - Shinkuu no Melancholy](https://osu.ppy.sh/beatmapsets/2152054#taiko/4534311). However, things quickly go south, as scattered misses give Poland a lead that they carry all the way to a break point. Poland then immediately move on to [Power Of Nature - BabeL ~Next Story~](https://osu.ppy.sh/beatmapsets/699595#taiko/1481393) and secure another point with a gamble on Turkey's Qualifiers weakness.

Reviewer: ```suggestion
Turkey opens by playing it safe, testing the waters with [KISIDA KYODAN & THE AKEBOSI ROCKETS - Shinkuu no Melancholy](https://osu.ppy.sh/beatmapsets/2152054#taiko/4534311). However, things quickly go south, as scattered misses give Poland a lead that they carry all the way to a break point. Team Poland then immediately moves on to [Power Of Nature - BabeL \~Next Story\~](https://osu.ppy.sh/beatmapsets/699595#taiko/1481393) and secures another point with a gamble on Turkey's Qualifier weakness.
```

> Turkey picks up a win with [Namiguru - Zunda Party Night](https://osu.ppy.sh/beatmapsets/1962520#taiko/4067448), but Poland continue to show that they mean business, quickly bringing the score to 1-4. Things are looking bad for Turkey, though Poland also seems to be feeling the pressure, as a narrow loss on [Street - Hypothesis](https://osu.ppy.sh/beatmapsets/2068643#taiko/4533909) gives Turkey a break point and a second wind. The audience rises from their seats, the Twitch chat copies their pastas and Turkey rides that wind, all the way to the first official RO32 Tiebreaker in Taiko World Cup history.

Reviewer: ```suggestion
Turkey picks up a win with [Namiguru - Zunda Party Night](https://osu.ppy.sh/beatmapsets/1962520#taiko/4067448), but the Polish continue to show that they mean business, quickly bringing the score to 1–4. Things are looking bad for Turkey, though Poland also seems to be feeling the pressure, as a narrow loss on [Street - Hypothesis](https://osu.ppy.sh/beatmapsets/2068643#taiko/4533909) gives Turkey a break point and a second wind. The audience rises from their seats, the Twitch chat copies their pastas, and Turkey rides that wind all the way to the first official Ro32 tiebreaker in osu!taiko World Cup history.
```

> [Yuuni - Frozen Blood](https://osu.ppy.sh/beatmapsets/2152052#taiko/4534309) fittingly sends chills down both teams' spines, and after a well-deserved timeout they begin their final battle. Turkey using 2 mods, Poland only 1. frukoyurdakul seems to have greatly enjoyed the timeout, as despite the added pressure of mods, he brings it home for his team with a 99.86% Hidden Full Combo.

Reviewer: ```suggestion
[Yuuni - Frozen Blood](https://osu.ppy.sh/beatmapsets/2152052#taiko/4534309) fittingly sends chills down both teams' spines, and after a well-deserved timeout they begin their final battle. Turkey using 2 mods, Poland only 1. frukoyurdakul seems to have greatly enjoyed the timeout, as despite the added pressure of mods, he brings it home for his team with a 99.86% Hidden Full Combo.
```

### ::{ flag=DE }:: Germany vs. ::{ flag=AR }:: Argentina

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2100431997&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Looking at Germany's roster for TWC 2024, you'd probably let out a wholeheartedly neutral "huh." Having finished 3rd in 2023, the previous German team decided that this was a good time to let the fresh blood shine with an entirely new roster. A roster that has landed them smack-bang in the middle of the seeding at #15, their Qualifiers results showing a clear preference for reading-based maps such as gimmicks and Hard Rock.

Reviewer: ```suggestion
Looking at Germany's roster for TWC 2024, you'd probably let out a wholeheartedly neutral "huh". Having finished 3rd in 2023, the previous German team decided that this was a good time to let the fresh blood shine with an entirely new roster. A roster that has landed them smack-bang in the middle of the seeding at #15, their Qualifiers results showing a clear preference for reading-based maps such as gimmicks and Hard Rock.
```

On the other side of the ring, Argentina also presents a completely new roster, putting them only a few steps behind Germany at #18. They share the love for reading maps, but lean more towards the Hidden side of the mod spectrum.

> Germany starts off the match with [siqlo - wholeheartedly](https://osu.ppy.sh/beatmapsets/2152039#taiko/4534272), and showcase their Gimmick prowess with a dominant first point. Argentina counters with a whole 6 mods on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) and lets the dust settle into a 1-1 tie. They say lightning never strikes twice, but Germany proceeds to win another 3 maps in a row and bring the score to 4-1. You see where this is going, right? Because Argentina then bounced back. All the way back to, you guessed it, yet another tiebreaker.

Reviewer: ```suggestion
The Germans starts the match off with [siqlo - wholeheartedly](https://osu.ppy.sh/beatmapsets/2152039#taiko/4534272), and showcases their *gimmick* prowess with a dominant first point. Argentina counters with a whole 6 mods on [Kotoha - Yuki wa Naniiro](https://osu.ppy.sh/beatmapsets/2078626#taiko/4352521) and lets the dust settle into a 1–1 tie. They say lightning never strikes twice, but Germany proceeds to win another 3 maps in a row and bring the score to 4–1. You see where this is going, right? Because Argentina then bounced back. All the way back to, you guessed it, yet another tiebreaker.
```

> Just like that, freshly written history has repeated itself. [Yuuni - Frozen Blood](https://osu.ppy.sh/beatmapsets/2152052#taiko/4534309) returns and Argentina show that they're in it to win it. They stack a whole 5 mods onto their team to counter Germany's 2. The map begins with Germany taking a small lead, though Argentina's mod advantage quickly proves useful as a 97.97% Hidden + Hard Rock score from [ZelLink](https://osu.ppy.sh/users/6752242) carries Argentina across the finish line. A spectacular comeback to wrap up a spectacular week of matches.

Reviewer: ```suggestion
Just like that, freshly written history has repeated itself. [Yuuni - Frozen Blood](https://osu.ppy.sh/beatmapsets/2152052#taiko/4534309) returns and the Argentines show that they're in it to win it. They stack a whole 5 mods onto their team to counter Germany's 2. The map begins with Germany taking a small lead, though Argentina's mod advantage quickly proves useful as a 97.97% Hidden + Hard Rock score from [ZelLink](https://osu.ppy.sh/users/6752242) carries Argentina across the finish line. A spectacular comeback to wrap up a spectacular week of matches.
```

---

> If you haven't, be sure to tune into this week's Round of 16 streams over on the [osulive](https://www.twitch.tv/osulive) Twitch channel, and if you are curious about every single detail regarding this World Cup, check out the [TWC 2024 wiki page](/wiki/Tournaments/TWC/2024)!

Reviewer: ```suggestion
If you haven't, be sure to tune into this week's Round of 16 streams over on the [osulive Twitch channel](https://www.twitch.tv/osulive), and if you are curious about every single detail regarding this World Cup including up-to-date schedules, check out the [TWC 2024 wiki page](/wiki/Tournaments/TWC/2024)!
```
